src/protocol/tx_format/input.md

```diff
@@ -19,20 +19,21 @@ Transaction is invalid if:
 
 ## InputCoin
 
-| name                  | type                         | description                                                            |
-|-----------------------|------------------------------|------------------------------------------------------------------------|
-| `txID`                | `byte[32]`                   | Hash of transaction.                                                   |
-| `outputIndex`         | `uint8`                      | Index of transaction output.                                           |
-| `owner`               | `byte[32]`                   | Owning address or predicate root.                                      |
-| `amount`              | `uint64`                     | Amount of coins.                                                       |
-| `asset_id`            | `byte[32]`                   | Asset ID of the coins.                                                 |
-| `txPointer`           | [TXPointer](./tx_pointer.md) | Points to the TX whose output is being spent.                          |
-| `witnessIndex`        | `uint8`                      | Index of witness that authorizes spending the coin.                    |
-| `maturity`            | `uint32`                     | UTXO being spent must have been created at least this many blocks ago. |
-| `predicateLength`     | `uint16`                     | Length of predicate, in instructions.                                  |
-| `predicateDataLength` | `uint16`                     | Length of predicate input data, in bytes.                              |
-| `predicate`           | `byte[]`                     | Predicate bytecode.                                                    |
-| `predicateData`       | `byte[]`                     | Predicate input data (parameters).                                     |
+| name                  | type                                   | description                                                            |
+|-----------------------|----------------------------------------|------------------------------------------------------------------------|
+| `txID`                | `byte[32]`                             | Hash of transaction or ID of originating message ID.                   |
```

Comment:
So in the case of a retried message, this is the tx id from the initial attempt to spend/use the message?

Reply:
In this scenario (message had no data and just has an amount) there isn't really a concept of retry. The amount would just move along to whatever outputs were specified. A retried message is just the same `InputMessage` until it is successfully spent.